client/wordpress-com.js

```diff
@@ -369,4 +369,12 @@ sections.push( {
 	secondary: true
 } );
 
+sections.push( {
+	name: 'domain-connect-authorize',
+	paths: [ '/domain-connect/authorize' ],
+	module: 'my-sites/upgrades/domain-management/domain-connect',
+	enableLoggedOut: false,
```

Comment:
When I visit the URL logged out, nothing happens. I'd expect to see the login window.